src/System.Net.Http/src/System/Net/Http/Unix/CurlHandler.cs

```diff
@@ -401,6 +407,30 @@ private void AddCredentialToCache(Uri serverUri, ulong authAvail, NetworkCredent
             }
         }
 
+        private void AddResponseCookies(Uri serverUri, HttpResponseMessage response)
+        {
+            if (!_useCookie)
+            {
+                return;
+            }
+
+            if (response.Headers.Contains(HttpKnownHeaderNames.SetCookie))
+            {
+                IEnumerable<string> cookieHeaders = response.Headers.GetValues(HttpKnownHeaderNames.SetCookie);
+                foreach (var cookieHeader in cookieHeaders)
+                {
+                    try
+                    {
+                        _cookieContainer.SetCookies(serverUri, cookieHeader);
+                    }
+                    catch (CookieException)
+                    {
+                        VerboseTrace("Malformed Cookie", "AddResponseCookies");
```

Comment:
You don't need to include "AddResponseCookies".  VerboseTrace picks up the name of the caller automatically.